include/revng/Yield/Internal/Graph.h

```diff
@@ -0,0 +1,67 @@
+#pragma once
```

Comment:
This commit says `Outline yield::Graph` but I only see new code. Why is that?

Reply:
It only OUTLINEs an interface. The actual usage of the new interface is introduced in the next commit.

Reply:
Uhm OK, as long as it compiles.

Reply:
It does, that's why the empty bodies and stuff are there.

Reply:
Closing this one, keeping the second one open for now.